AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -360,6 +360,7 @@ modivo.*#%#//scriptlet('set-cookie', '__MODIVO__cookies_policy', 'true')
 joybuy.de#%#//scriptlet('trusted-set-cookie', 'cookie_consent', '%7B%22e%22%3A1%2C%22f%22%3A0%2C%22a%22%3A0%2C%22t%22%3A0%7D')
 goodsport.pl#%#//scriptlet('trusted-click-element', '#cc--main button[class="c-bn"], #cc--main button#s-rall-bn', '!cookie:cc_cookie')
 premiumtech.pl,tri-elektro.pl,yoer.pl#%#//scriptlet('trusted-click-element', '#cc--main button.c-bn.c_link, #cc--main button#s-rall-bn', '!cookie:cc_cookie')
+npistanbul.com##body > #__next > div[data-overlay-container] > div.bg-third
```

Comment:
I missed the PR, but we have more common domain-specific rule

```adblock
##.fixed.bottom-0
```

Now just remove your rule.